### PR TITLE
POC new skeleton text component

### PR DESCRIPTION
**Pull request by Oluane**
## Description

This PR tries to improve the skeleton component to replace not all components but just "text" line til the page is populated with data. 

Changes have been made on `/security`, `/account/logins/[type]/[id]` and `/account/logins/[type]/[id]/update`. On the last two, the identity text has a skeleton component and buttons are just not displayed til data is available. 

A little rework of the style (mostly color) of skeleton components has also been made.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [x] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-gh8q7a [feature] - Improve skeleton component to just replace text and not box](https://app.clickup.com/t/gh8q7a)

File: src/components/forms/update-identity-form.tsx [hunk 1]
```diff
@@ -20,121 +21,133 @@ import { fetchJson } from "@src/utils/fetch-json";
...
 
 const UpdateIdentityForm: React.FC<{
-  currentIdentity: Identity;
-}> = ({ currentIdentity }) => {
+  data?: { identity: Identity };
+}> = ({ data }) => {
   const [identity, setIdentity] = React.useState<InMemoryTemporaryIdentity>({
     value: "",
-    type: currentIdentity.type,
+    type: data ? data.identity.type : IdentityTypes.EMAIL,
```
Comment (Oluane) [inline src/components/forms/update-identity-form.tsx:28]: Not very proud of this one. If you've got any better idea, I would appreciate 🙂  @Fuzznimp

Comment (Fuzznimp) [reply src/components/forms/update-identity-form.tsx:28]: Well, except if we add a `UNKNOWN` enum entry, I guess we can't do better :/.

File: src/components/pages/security/security.tsx [hunk 2]
```diff
@@ -19,18 +19,16 @@ const Security: React.FC = () => {
...
-          <SecurityLink href="/account/security/update">
-            <TextBox>
-              {data.isPasswordSet ? "Update" : "Set"} your password
-            </TextBox>
-            <RightChevron />
-          </SecurityLink>
-        </SectionBox>
-      )}
+      <SectionBox>
+        <SecurityLink href={!data ? "#" : "/account/security/update"}>
```
Comment (Fuzznimp) [inline src/components/pages/security/security.tsx:23]: Would an empty string works here?

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**